src/hooks/AllPaymentHooks.res

```diff
@@ -402,6 +408,45 @@ let useRedirectHook = () => {
             },
           )
         }
+      | "third_party_sdk_session_token" => {
+        // TODO: add event loggers for analytics
+          let defaultSessionToken = {
+            wallet_name: "",
+            open_banking_session_token: "",
+          }
+
+          let session_token = switch nextAction {
+          | Some(nextAction) => nextAction.session_token
+          | None => Some(defaultSessionToken)
+          }
+
+          switch session_token {
```

Comment:
remove this with an if statement as it is binary in nature ( session_token presence is either true or false)

Reply:
`session_token` is `record` type but we can only use `boolean` with `if` statements. Can you please explain this a bit more.

Reply:
if session_token->Option.isSome { 
// ...
}
use this, as we are not doing anything if session_token is None